### PR TITLE
Removed print calls and fixed pep8 warnings in touched files.

### PR DESCRIPTION
**Pull request by towermagi**
Replaced all print statements with logger.debug or logger.info calls. If the print statement occurred in a self.verbose check then debug was used otherwise info was used.

Also fixed and pep8 warnings in the modified files including:
* missing or to much white space
* lines that were to long
* missing or to many blank lines
* two # noinspection PyMethodMyBeStatic comments
* exception to broad clause, by logging a debug of the exception rather than passing
* variable value may not be used
* finally added botocore to the requirements file for pep8 even though boto3 implies it

### CODE REVIEW

**Comment by towermagi** [comment]
I had the same two test fail locally it appears cerberus.fake.com is no longer valid domain?
I also installed the changes locally and verified they do remove the print's I was seeing in our project.

File: cerberus/aws_auth.py [hunk 1]
```diff
@@ -68,7 +69,7 @@ def get_token(self):
         token = resp.json()['client_token']
         iam_principal_arn = resp.json()['metadata']['aws_iam_principal_arn']
         if self.verbose:
-            print('Successfully authenticated with Cerberus as {}'.format(iam_principal_arn), file=sys.stderr)
+            logger.debug('Successfully authenticated with Cerberus as {}'.format(iam_principal_arn), file=sys.stderr)
```
Comment (fieldju) [inline cerberus/aws_auth.py]: We want this to be info logging please, we spend a lot of time on slack supporting this client and would rather have it be noisy than deal with telling customers to enable debug logging over and over.

File: cerberus/client.py [hunk 2]
```diff
@@ -76,11 +80,11 @@ def _set_token(self):
...
             self.token = os.environ['CERBERUS_TOKEN']
             if self.verbose:
-                print("Overriding Cerberus token with environment variable.", file=sys.stderr)
+                logger.debug("Overriding Cerberus token with environment variable.", file=sys.stderr)
             logger.info("Overriding Cerberus token with environment variable.")
             return
-        except:
-            pass
+        except Exception as e:
+            logger.debug(str(e))
```
Comment (fieldju) [inline cerberus/client.py]: Did this print the stack trace before and will now debug log it if debug is enabled?

- [ ] Ensure that the stack trace is still logged by default, if that was the previous behavior.

File: cerberus/aws_auth.py (hunk 1 above)
Comment (fieldju) [reply cerberus/aws_auth.py]: - [ ] make this be logged by default

**Comment by mayitbeegh** [comment]
@towermagi you can merge it now

**Comment by towermagi** [comment]
> @towermagi you can merge it now

I do not possess the write access needed to merge the request.

**Comment by mayitbeegh** [comment]
> > @towermagi you can merge it now
> 
> I do not possess the write access needed to merge the request.

My bad. It's published.

**Pull request merged**